src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,127 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.Logger;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    private static final Logger log = Logger.getLogger();
+
+    protected FeedbackSessionAttributes extractFeedbackSessionData(boolean isCreating) {
```

Comment:
* Could you add a short description of what this method does? As its quite long.
* isCreating: rename to e.g isCreatingSession?